src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -612,16 +640,16 @@ public VariableScope[] GetVariableScopes(int stackFrameId)
                 new VariableContainerDetails(this.nextVariableId++, "Scope: " + scope);
             this.variables.Add(scopeVariableContainer);
 
-            var results = await this.powerShellContext.ExecuteCommand<PSVariable>(psCommand, sendErrorToHost: false);
+            var results = await this.powerShellContext.ExecuteCommand<PSObject>(psCommand, sendErrorToHost: false);
```

Comment:
Remote/attached runspaces return some concrete type that looks like a PSVariable but isn't the same actual type, thus I can't pull them back out as PSVariable.  Same thing happened with the CallStackFrames.  Not sure why it was done this way, but probably because remote sessions were added later and the types weren't designed to be remotable in the beginning.